nsdb-http/src/test/scala/io/radicalbit/nsdb/actor/FakeReadCoordinator.scala

```diff
@@ -57,7 +57,7 @@ class FakeReadCoordinator extends Actor {
         Schema("metric", bits.head).getOrElse(Schema("metric", Map.empty[String, SchemaField]))) match {
         case Right(_) =>
           val e = statement.condition.get.expression.asInstanceOf[RangeExpression[Long]]
```

Comment:
this should be `RangeExpression[AbsoulteComparisonValue[Long]`

Reply:
As wee see, it is correct because `RangeExpression` is 

```
case class RangeExpression[T](dimension: String, value1: ComparisonValue[T], value2: ComparisonValue[T])
    extends Expression
```

Reply:
yes. Sorry for this bad comment